packages/terra-arrange/docs/Arrange.md

```diff
@@ -0,0 +1,9 @@
+| Prop Name | Type | Is Required | Default Value | Description |
```

Comment:
I think we only want to in the markdown table when we publish a package. I think we should add the each prop-table to our gitignore. What are your thoughts?